src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -68,38 +68,23 @@ bool Terminal::SetTextRgbColor(COLORREF color, bool foreground) noexcept
 bool Terminal::BoldText(bool boldOn) noexcept
 {
     TextAttribute attrs = _buffer->GetCurrentAttributes();
-    if (boldOn)
-    {
-        attrs.Embolden();
-    }
-    else
-    {
-        attrs.Debolden();
-    }

```

Comment:
Mildly sad to see the terminology of "embolden" and "debolden" go. It was always amusing when I came across it.